lib/tapioca/commands/todo.rb

```diff
@@ -6,6 +6,15 @@ module Commands
     class Todo < CommandWithoutTracker
       include SorbetHelper
 
+      DEPRECATION_MESSAGE = T.let(<<~DEPRECATION, String)
+        The `todo` command is deprecated and will be removed in a future release.
+
+        If your project is still missing type definitions for constants, try the following:
+        - Regenerate gem RBIs by running `bin/tapioca gem --all` and `bin/tapioca annotations`
```

Comment:
From my understanding, these steps are best used together and in this order. If that's correct, should we make them an ordered list instead?

Reply:
Yes this is an excellent point!